client/src/components/Header/style.js

```diff
@@ -0,0 +1,26 @@
+import {StyleSheet} from 'react-native';
+
+const styles = StyleSheet.create({
+  bottom: {
+    backgroundColor: '#24292E',
+    left: 0,
+    right: 0,
+    bottom: 0,
```

Comment:
You don't need these, we'll just put the header on top of any screen every time so we don't need these positions. You can remove all of this style and move the background color to the container instead.